defaults/main.yml

```diff
@@ -0,0 +1,317 @@
+mattermost_Service_SessionLengthSSOInDays: 30
+mattermost_Service_SessionCacheInMinutes: 10
+mattermost_Service_SessionIdleTimeoutInMinutes: 0
+mattermost_Service_WebsocketSecurePort: 443
+mattermost_Service_WebsocketPort: 80
+mattermost_Service_WebserverMode: "gzip"
+mattermost_Service_EnableCustomEmoji: "false"
+mattermost_Service_EnableEmojiPicker: "true"
+mattermost_Service_RestrictCustomEmojiCreation: "all"
+mattermost_Service_RestrictPostDelete: "all"
+mattermost_Service_AllowEditPost: "always"
+mattermost_Service_PostEditTimeLimit: 300
+mattermost_Service_TimeBetweenUserTypingUpdatesMilliseconds: 5000
+mattermost_Service_EnablePostSearch: "true"
+mattermost_Service_EnableUserTypingMessages: "true"
+mattermost_Service_EnableChannelViewedMessages: "true"
+mattermost_Service_EnableUserStatuses: "true"
+mattermost_Service_ClusterLogTimeoutMilliseconds: 2000
+
+mattermost_Team_SiteName: "Mattermost"
+mattermost_Team_MaxUsersPerTeam: 500
+mattermost_Team_EnableTeamCreation: "false"
+mattermost_Team_EnableUserCreation: "true"
+mattermost_Team_EnableOpenServer: "false"
+mattermost_Team_RestrictCreationToDomains: "true"
+mattermost_Team_EnableCustomBrand: "true"
+mattermost_Team_CustomBrandText: ""
+mattermost_Team_CustomDescriptionText: ""
+mattermost_Team_RestrictDirectMessage: "any"
+mattermost_Team_RestrictTeamInvite: "all"
+mattermost_Team_RestrictPublicChannelManagement: "all"
+mattermost_Team_RestrictPrivateChannelManagement: "all"
+mattermost_Team_RestrictPublicChannelCreation: "all"
+mattermost_Team_RestrictPrivateChannelCreation: "all"
+mattermost_Team_RestrictPublicChannelDeletion: "all"
+mattermost_Team_RestrictPrivateChannelDeletion: "all"
+mattermost_Team_RestrictPrivateChannelManageMembers: "all"
+mattermost_Team_EnableXToLeaveChannelsFromLHS: "false"
+mattermost_Team_UserStatusAwayTimeout: 300
+mattermost_Team_MaxChannelsPerTeam: 2000
+mattermost_Team_MaxNotificationsPerChannel: 1000
+mattermost_Team_EnableConfirmNotificationsToChannel: "true"
+mattermost_Team_TeammateNameDisplay: "username"
+mattermost_Team_ExperimentalTownSquareIsReadOnly: "false"
+
+mattermost_ClientRequirements_AndroidLatestVersion: ""
+mattermost_ClientRequirements_AndroidMinVersion: ""
+mattermost_ClientRequirements_DesktopLatestVersion: ""
+mattermost_ClientRequirements_DesktopMinVersion: ""
+mattermost_ClientRequirements_IosLatestVersion: ""
+mattermost_ClientRequirements_IosMinVersion: ""
+
+mattermost_Sql_DriverName: "postgres"
+#Sql_DataSource: "postgres://{{ db_user }}:{{ db_password }}@127.0.0.1:5432/{{ db_name }}?sslmode=disable&connect_timeout=10"
+#Sql_DataSource is implemented with the template mattermost-config.json.j2 - setting it here may have strange consequences
+mattermost_Sql_DataSourceReplicas: []
+mattermost_Sql_DataSourceSearchReplicas: []
+mattermost_Sql_MaxIdleConns: 20
+mattermost_Sql_MaxOpenConns: 300
+mattermost_Sql_Trace: "false"
+mattermost_Sql_AtRestEncryptKey: "jksksj85pzbomhs3ockhrzynpgh8ox34"
```

Comment:
please mention important stuff like that, maybe we don't want the same key across all instances